account_asset_batch_compute/wizards/account_asset_compute.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016-2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class AccountAssetCompute(models.TransientModel):
+
+    _inherit = "account.asset.compute"
+
+    batch_processing = fields.Boolean()
+
+    def asset_compute(self):
+        self.ensure_one()
+        if not self.batch_processing:
+            return super(AccountAssetCompute, self).asset_compute()
```

Comment:
Apply PEP 3135 (28-Apr-2007) to "super()."

Reply:
Done.